sdk/storage/azure-storage-blob/azure/storage/blob/_quick_query_helper.py

```diff
@@ -130,7 +129,7 @@ class QuickQueryStreamer(object):
     File-like streaming iterator.
     """
 
-    def __init__(self, generator):
+    def __init__(self, generator) -> None:
```

Comment:
Remove this unless you want to type this whole class. It is not exported though so fine to leave out for now I guess.